### DISCUSSION TITLE
clone/duplicate a configuarion

### DISCUSSION THREAD

**Discussion by dulhaver** [SchoofsKelvin/vscode-sshfs#352] [Q&A]
If you wanted to use an SSH connection to the same remote with different wording directories, but apart from this with the exact same settings, it would be helpful to duplicate a configuration.

I do not see an option for doing this anywhere though. How can I do that?

P.S. I hope this is the right location for such a question. If not ... should I use the issues for that instead ?

**Comment by SchoofsKelvin** [comment]
Duplicate of #268
